App/VSTS/FollowButton.tsx

```diff
@@ -0,0 +1,66 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { changeFollow } from '../actionsEZ';
+import { connect } from 'react-redux';
+import { FollowStateTypes } from '../statesEZ';
+
+let buttonText: string = '';
+let isDisabled: boolean = false;
+let iconType: string = 'ms-Icon ms-Icon--eye';
+
+interface IFollowButtonProps {
+  dispatch?: any;
+  followState?: FollowStateTypes;
+}
+
+function mapStateToProps(state: any): IFollowButtonProps {
+  return {
+    followState: state.quickActionState.followState,
+  };
+}
+
+@connect(mapStateToProps)
+export class FollowButton extends React.Component<IFollowButtonProps, {}> {
+  public render(): React.ReactElement<Provider> {
+    console.log('got to followbutton, followstate: ' + this.props.followState);
+    switch (this.props.followState) {
```

Comment:
Not fixed. The interally used enum is named something more informative, thus the switch to the less clear terms used in the UI for the VSTS web form.